src/Umbraco.Web.Common/Extensions/UmbracoBuilderExtensions.cs

```diff
@@ -135,20 +142,20 @@ public static IUmbracoBuilder AddUmbracoCore(this IUmbracoBuilder builder)
             }
 
                 );
-            builder.Services.AddUnique<IAppPolicyCache>(factory => factory.GetRequiredService<AppCaches>().RuntimeCache);
-            builder.Services.AddUnique<IRequestCache>(factory => factory.GetRequiredService<AppCaches>().RequestCache);
+            builder.Services.AddUnique(factory => factory.GetRequiredService<AppCaches>().RuntimeCache);
```

Comment:
`AddUnique(factory)` is on my kill list. Horrible method which uses the service provider unnecessarily.